joomla/joomla-cms
# Fix W3C Warning for breadcrumbs: role="navigation" is unnecessary for element nav

### Summary of Changes
The breadcrumbs have the ARIA role="navigation", but this is unnecessary for nav elements. Also added a class to the breadcrumbs nav element for easy styling.

### Testing Instructions
Look at the code of the breadcrumbs and see if the the role element is there. Apply the patch and check if the role element is gone and the class added.

### Actual result BEFORE applying this Pull Request
breadcrumbs nav has role="navigation"

### Expected result AFTER applying this Pull Request
breadcrumbs nav has no role="navigation" and has class="breadcrumbs-container"

### Documentation Changes Required
No
